internal/directives/zz_config_types.go

```diff
@@ -436,3 +450,12 @@ const (
 	Github Provider = "github"
 	Gitlab Provider = "gitlab"
 )
+
+// The new value for the specified key.
+type Value struct {
+	AnythingArray []interface{}
+	AnythingMap   map[string]interface{}
+	Bool          bool
+	Double        int64
+	String        string
+}
```

Comment:
This generated code is quirky. Again... the tool we're using for this bit of codegen is temperamental. It's worth playing with the schema more to see if we can't coerce this into something that will work better.

`Double int64` is obviously wrong.

Reply:
[This](https://github.com/akuity/kargo/pull/3151#discussion_r1889083798) will help.